dbf-reader/src/main/java/org/jamel/dbf/structure/DbfHeader.java

```diff
@@ -102,4 +108,22 @@ public int getNumberOfRecords() {
     public short getRecordLength() {
         return recordLength;
     }
+
+    int getFieldIndex(String fieldName) {
+        if (fieldIndexesByNames == null) {
+            initFieldIndexesByNames();
+        }
+        if (!fieldIndexesByNames.containsKey(fieldName)) {
+            throw new DbfException(format("Field \"%s\" does not exist", fieldName));
```

Comment:
It is better to return `-1` here if field name was not found and handle this special value depending on `DbfRow` logic.